cli/cmd/up/up.go

```diff
@@ -151,7 +151,15 @@ func (u *Up) up(content string, answers map[string]string, s *riov1.Stack, c *cl
 		knowngvks = s.Spec.AdditionalGroupVersionKinds
 	}
 
-	err = c.Apply.WithListerNamespace(c.GetSetNamespace()).WithDefaultNamespace(c.GetSetNamespace()).WithOwner(s).WithSetOwnerReference(true, true).WithGVK(knowngvks...).WithDynamicLookup().ApplyObjects(objs...)
+	err = c.Apply.
+		WithListerNamespace(c.GetSetNamespace()).
+		WithDefaultNamespace(c.GetSetNamespace()).
+		WithOwner(s).
+		WithSetOwnerReference(true, true).
+		WithGVK(knowngvks...).
+		WithRestrictClusterScoped().
```

Comment:
👍  This means riofiles cannot have cluster scoped resources correct?

Reply:
correct